program/find-average-of-numbers-by-recursion/FindAverageOfNumbersByRecursion.ts

```diff
@@ -0,0 +1,12 @@
+function average_of_numbers(arr: any, length: number, i: number): any {
+    if (i === length - 1) {
+      return arr[i];
+    }
+    else if (i === 0) return (arr[i] + average_of_numbers(arr, length, i + 1)) / length;
+  
+    else return arr[i] + average_of_numbers(arr, length, i + 1);
+  }
+  
+  let array = [1, 2, 3, 4, 5];
+  let i = 0;
+  console.log(average_of_numbers(array, array.length, i));
```

Comment:
Can we get only the array as an input in the arguments and automate the rest based on the input .
Here you are expecting the user to provide array.length and i as an argument .

Reply:
That's a good point 👍🏻 

We can create a separate `good first issue` for this as this PR is already merged